Incompatible minimum golang version declared in go.mod

Change #280 depends on Value.Grow from the reflect package, this method was added in [go 1.20](https://tip.golang.org/doc/go1.20)
> The new [Value.Grow](https://tip.golang.org/pkg/reflect/#Value.Grow) method extends a slice to guarantee space for another n elements.

The minimum go version required by the go-openapi runtime module is defined as 1.19 in [go.mod](https://github.com/go-openapi/runtime/blob/master/go.mod#L38).

This results in an error using v0.27 as dependency in a project compiled with go 1.19:
```
/go/pkg/mod/github.com/go-openapi/runtime@v0.27.0/csv.go:127:7: v.Grow undefined (type reflect.Value has no field or method Grow)
```

Perhaps the minimum required go version for go-openapi runtime should be bumped to at least 1.20 in go.mod. I see the tests run with the latest go version (1.21 atm), not sure if you want to go as far as to tests with different go versions (starting from the minimum required version as defined in go.mod) to spot this kind of dependencies on newer go versions.

--- a/.github/workflows/go-test.yml
+++ b/.github/workflows/go-test.yml
@@ -27,7 +27,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        go_version: ['oldstable', 'stable' ]
+        go_version: ['oldstable', 'stable', '1.19' ]
 
     steps:
     - name: Run unit tests

--- a/csv.go
+++ b/csv.go
@@ -124,10 +124,16 @@ func CSVConsumer(opts ...CSVOpt) Consumer {
 				if err := pipeCSV(csvWriter, csvReader, o); err != nil {
 					return err
 				}
-				v.Grow(len(csvWriter.records))
-				v.SetCap(len(csvWriter.records)) // in case Grow was unnessary, trim down the capacity
-				v.SetLen(len(csvWriter.records))
-				reflect.Copy(v, reflect.ValueOf(csvWriter.records))
+
+				/*
+					// with go1.20:
+					v.Grow(len(csvWriter.records))
+					v.SetCap(len(csvWriter.records)) // in case Grow was unnessary, trim down the capacity
+					v.SetLen(len(csvWriter.records))
+					reflect.Copy(v, reflect.ValueOf(csvWriter.records))
+				*/
+				v.SetLen(0)
+				v.Set(reflect.AppendSlice(v, reflect.ValueOf(csvWriter.records)))
 
 				return nil
 

